apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.ts

```diff
@@ -347,8 +370,10 @@ router.post(
           req.body.tag_for_manual_grading,
           res.locals.authn_user.user_id,
         );
+        // TODO: update this handler when rubric settings are modified differently
```

Comment:
This comment doesn't mean anything to me and doesn't help me evaluate if this needs to change before this lands. For me and whoever ends up working on this, can you spell out what's broken/deficient/etc. with the current code?

Reply:
It is a server-side handler that can be converted to a client-side handler to avoid a page refresh.